src/Tests/dotnet-vstest.Tests/VSTestTests.cs

```diff
@@ -124,6 +124,37 @@ public void ItShouldSetDotnetRootToLocationOfDotnetExecutable()
             result.StartInfo.EnvironmentVariables[dotnetRoot].Should().Be(Path.GetDirectoryName(dotnet));
         }
 
+        [Fact]
+        public void ItShouldAcceptMultipleLoggers() {
+            var testProjectDirectory = this.CopyAndRestoreVSTestDotNetCoreTestApp();
+
+            var configuration = Environment.GetEnvironmentVariable("CONFIGURATION") ?? "Debug";
+
+            new BuildCommand(Log, testProjectDirectory)
+                .Execute()
+                .Should().Pass();
+
+            var outputDll = Path.Combine(testProjectDirectory, "bin", configuration, ToolsetInfo.CurrentTargetFramework, "VSTestTestRunParameters.dll");
+
+            // Call test
+            CommandResult result = new DotnetVSTestCommand(Log)
+                                        .Execute(new[] {
+                                            outputDll,
+                                            "--logger:console;verbosity=normal",
+                                            "--logger:trx",
+                                            "--",
+                                            "TestRunParameters.Parameter(name=\"myParam\",",
+                                            "value=\"value\")",
+                                            "TestRunParameters.Parameter(name=\"myParam2\",",
+                                            "value=\"value",
+                                            "with",
+                                            "space\")"
+                                        });
+
+            // Verify
+            result.ExitCode.Should().Be(0, $"Should have executed successfully, but got: {result.StdOut}");
```

Comment:
could you assert that the console output is the expected one(you can check the assertion above) and that the trx file is created inside the `TestResults` folder?

Reply:
Can you add a test without any logger?

Reply:
Added both cases, thanks for the pointers. for the file-existence check in particular I had to name the TRX file so I could verify for this specific test the file was created.

Reply:
good pls add the assertion also to the `no logger test`
```cs
if (!TestContext.IsLocalized())
            {
                result.StdOut.Should().NotMatch("The test run parameter argument '*' is invalid.");
                result.StdOut.Should().Contain("Total tests: 1");
                result.StdOut.Should().Contain("Passed: 1");
                result.StdOut.Should().Contain("Passed VSTestTestRunParameters");
            }
```

An I think we're good to go.

Reply:
oh - I wasn't sure that without the console logger we'd even have input!

after I add this, the servicing template will need added

Reply:
Good question it should be there I suppose 😃 but now you told me I'm not sure enough let's try(I don't recall if we add it as default logger or console is special).

Reply:
yeah, that causes failures:

```
[xUnit.net 00:00:16.38]     Microsoft.DotNet.Cli.VSTest.Tests.VSTestTests.ItShouldAcceptNoLoggers [FAIL]
  Failed Microsoft.DotNet.Cli.VSTest.Tests.VSTestTests.ItShouldAcceptNoLoggers [4 s]
  Error Message:
   Expected string "Microsoft (R) Test Execution Command Line Tool Version 17.2.0 (x64)\r\nCopyright (c) Microsoft Corporation.  All rights reserved.\r\n\r\nStarting test execution, please wait...\r\nA total of 1 test files matched the specified pattern.\r\n\r\nPassed!  - Failed:     0, Passed:     1, Skipped:     0, Total:     1, Duration: 20 ms - VSTestTestRunParameters.dll (net6.0)" to contain "Total tests: 1".
  Stack Trace:
     at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.StringAssertions.Contain(String expected, String because, Object[] becauseArgs)
   at Microsoft.DotNet.Cli.VSTest.Tests.VSTestTests.ItShouldAcceptNoLoggers() in D:\Code\dotnet-sdk\src\Tests\dotnet-vstest.Tests\VSTestTests.cs:line 199
  Standard Output Messages:
 > D:\Code\dotnet-sdk\artifacts\bin\redist\Debug\dotnet\dotnet.exe msbuild /t:Restore D:\Code\dotnet-sdk\artifacts\tmp\Debug\ItShouldAccep---5A910974\VSTestTestRunParameters.csproj
 Microsoft (R) Build Engine version 17.2.2+038f9bae9 for .NET
 Copyright (C) Microsoft Corporation. All rights reserved.

   Determining projects to restore...
   Restored D:\Code\dotnet-sdk\artifacts\tmp\Debug\ItShouldAccep---5A910974\VSTestTestRunParameters.csproj (in 557 ms).
 > D:\Code\dotnet-sdk\artifacts\bin\redist\Debug\dotnet\dotnet.exe msbuild /t:Build D:\Code\dotnet-sdk\artifacts\tmp\Debug\ItShouldAccep---5A910974\VSTestTestRunParameters.csproj /restore
 Microsoft (R) Build Engine version 17.2.2+038f9bae9 for .NET
 Copyright (C) Microsoft Corporation. All rights reserved.

   Determining projects to restore...
   All projects are up-to-date for restore.
   VSTestTestRunParameters -> D:\Code\dotnet-sdk\artifacts\tmp\Debug\ItShouldAccep---5A910974\bin\Debug\net6.0\VSTestTestRunParameters.dll
 > D:\Code\dotnet-sdk\artifacts\bin\redist\Debug\dotnet\dotnet.exe vstest D:\Code\dotnet-sdk\artifacts\tmp\Debug\ItShouldAccep---5A910974\bin\Debug\net6.0\VSTestTestRunParameters.dll -- TestRunParameters.Parameter(name=\"myParam\", value=\"value\") TestRunParameters.Parameter(name=\"myParam2\", value=\"value with space\")
 Microsoft (R) Test Execution Command Line Tool Version 17.2.0 (x64)
 Copyright (c) Microsoft Corporation.  All rights reserved.

 Starting test execution, please wait...
 A total of 1 test files matched the specified pattern.

 Passed!  - Failed:     0, Passed:     1, Skipped:     0, Total:     1, Duration: 20 ms - VSTestTestRunParameters.dll (net6.0)
 ```

so I'll revert this particular change.